### ISSUE TITLE
Get rid of FunctionsPrint*

### ISSUE THREAD

**Issue by magicsunday** [fisharebest/webtrees#1584]
Currently a lot of HTML is hidden inside the "FunctionsPrint*" classes, making it hard to understand whats going on in these methods and difficult to write a custom theme which should not rely on the hard coded HTML structure these classes provide.

Also a lot of the code is used by different views and should be splitted somehow for easier reusability and maintainability.

**Comment by fisharebest** [comment]
I agree 100%

You will see that:

* HTML is being moved to the templates folder `resources/views/...`
* themes can override all the template files

Also, see #1576 - I want to replace similar "HTML" functions in the themes with template files.

**Comment by magicsunday** [comment]
Yeah, I already noticed the templates folder, which is a first step in the right direction. 👍
